### DISCUSSION TITLE
Duda ejercicio  2.1 Largo de cadenas

### DISCUSSION THREAD

**Discussion by NicoHermosilla** [INGCOM-UNRN-P1/cursada-2023#120] [Preguntas y respuestas] [answered]
Quisiera saber si lo planteado en mi ejercicio está bien o estoy confundiendo algunas cosas, esto viendo que lo que converso al final de la clase me genero confusión
https://github.com/INGCOM-UNRN-P1/tp3-2023-NicoHermosilla

**Comment by martinvilu** [comment, marked as answer]
Uno solo `return` por función

**Reply by martinvilu** [reply]
¿En que se diferencian `i` y `largo`?

**Reply by martinvilu** [reply]
Se ve bien, ¿que confusión apareció?

**Reply by NicoHermosilla** [reply]
Se diferencian en uno sirve para recorrer la cadena(osea i) , mientras que el otro(osea largo) sirve para contabilizar el largo de la cadena ingresada.
Mi confusion partida de que creia que habia planteado mal la funcion pero al parecer y ahora que lo pienso mejor cumple con lo requiero.
Pd: ahi cambie los return que habian por uno solo.
Gracias

**Reply by martinvilu** [reply]
> Se diferencian en uno sirve para recorrer la cadena(osea i) , mientras que el otro(osea largo) sirve para contabilizar el largo de la cadena ingresada. 

![7xjxl3](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/56625/8a2c9a96-37ac-45e0-8e6e-bfa133a70244)

**Reply by NicoHermosilla** [reply]
https://github.com/INGCOM-UNRN-P1/tp3-2023-NicoHermosilla
Despues de ver la imagen me di cuenta de que estaba usando alguna de las 2 de mas, pero ahora tengo problemas para implementar un caso en el que la cadena sea vacia ya que no puedo no poner nada cuando pido que se ingrese la cadena.
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/105023590/7f20ded9-0b2b-4b50-83bb-14d70fd32d62)
